source/_components/remember_the_milk.markdown

```diff
@@ -0,0 +1,51 @@
+---
+layout: page
+title: "Remember The Milk"
+description: "Instructions on how to use Remember The Milk with Home Assistant."
+date: 2017-10-10 10:10
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
+ha_category: Other
+ha_release: 0.23
+---
+
+The `Remember The Milk` component allows you to create tasks in [Remember The Milk](https://www.rememberthemilk.com) from Home Assistant. You can use this if you want Home Assistant to send you a task that you should not forget, e.g. water the plants.
+
+# Setup
```

Comment:
Please make the heading linkable -> https://home-assistant.io/developers/documentation/standards/#general-documentation

Reply:
OK